.github/workflows/doc.yml

```diff
@@ -3,21 +3,25 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
+
```

Comment:
let's add inputs like api-version

Reply:
so we will be able to create new version

Reply:
Done!